README.md

```diff
@@ -6,6 +6,7 @@
 
 - clone repo
 - `npm i`
+- `npm run build`
```

Comment:
not sure that it's needed to run before local start
why would you need to run it?

Reply:
Right, works without. I removed it.